src/Felix/Instances/Function/Lift.agda

```diff
@@ -5,26 +5,32 @@ open import Level
 module Felix.Instances.Function.Lift (a b : Level) where
 
 open Lift
-open import Data.Product using (_,_)
+open import Data.Product using (_,_; uncurry)
+open import Function using (_∘_)
 
-open import Felix.Homomorphism hiding (refl)
+open import Felix.Homomorphism
 open import Felix.Object
-
-private module F {ℓ} where open import Felix.Instances.Function ℓ public
-open F
+import Felix.Instances.Function
+open module F {ℓ} = Felix.Instances.Function ℓ
```

Comment:
Nice. I had never seen this syntax ("`open module ... = ... `") before.

Reply:
I've learned about in the [zulip](https://agda.zulipchat.com/#narrow/stream/259644-newcomers/topic/.E2.9C.94.20Instantiating.20modules.20polymorphically.3F/near/366625295)

Reply:
Nice. Thanks for the pointer.